requirements.md

```diff
@@ -51,9 +51,12 @@ As we identify the requirements and constraints, a number of key contributors wi
   - [Quay][quay] - Hank Donnay <hdonnay@redhat.com>
   - [IBM Cloud Container Registry (icr)][icr]
 - Registry Vendors, Projects & Products
-  - [Docker Trusted Registry][docker-dtr]
+  - [Mirantis Secure Registry (msr, formerly Docker Trusted Registry)][msr]
   - [Harbor][harbor]
   - [JFrog Artifactory][jfrog]
+- Controllers, Runtimes & Engines
+  - [Mirantis Container Runtime (mcr, formerly Docker Engine – Enterprise)][mcr]
```

Comment:
Nit: please change `mcr` to possibly `mirantis-runtime`

Reply:
Thanks for the suggestion, i've removed `mcr` in my latest push.